app/models/deletion.rb

```diff
@@ -30,7 +30,7 @@ def record_metadata
   end
 
   def expire_api_memcached
-    ["deps/v1/#{rubygem}", "info/#{rubygem}", "versions", "names"].each do |key|
+    ["deps/v1/#{rubygem}", "versions", "names"].each do |key|
```

Comment:
where is it that the cache gets set? `set_yanked_info_checksum` only sets the `yanekd_info_checksum` in the DB, which won't be used since the version won't be fully yanked yet